app/src/main/java/net/programmierecke/radiodroid2/adapters/ItemAdapterStation.java

```diff
@@ -174,6 +176,15 @@ public void enableItemRemoval(RecyclerView recyclerView) {
         }
     }
 
+    public void enableItemMoveAndRemoval(RecyclerView recyclerView) {
+        if (!supportsStationRemoval) {
+            supportsStationRemoval = true;
+
+            RecyclerItemSwipeHelper<StationViewHolder> swipeAndMoveHelper = new RecyclerItemMoveAndSwipeHelper<>(ItemTouchHelper.UP + ItemTouchHelper.DOWN, ItemTouchHelper.LEFT +ItemTouchHelper.RIGHT, this);
```

Comment:
Unchecked assigment

Reply:
OK, this should probably be `RecyclerItemMoveAndSwipeHelper<StationViewHolder>`